src/extras/curves/CatmullRomCurve3.js

```diff
@@ -79,7 +79,9 @@ function CubicPoly() {
 //
 
 const tmp = /*@__PURE__*/ new Vector3();
-const px = /*@__PURE__*/ new CubicPoly(), /*@__PURE__*/ py = new CubicPoly(), /*@__PURE__*/ pz = new CubicPoly();
+const px = /*@__PURE__*/ new CubicPoly(),
```

Comment:
It's okay here but not make a general rule out of this. There are use cases where having everything in a single line improves the readability of the entire file e.g.

https://github.com/mrdoob/three.js/blob/de17358232c3cc45bb2df0a5a64068cbcdeaa2f8/src/renderers/webgl/WebGLUtils.js#L1